src/sql/src/session/vars.rs

```diff
@@ -2200,6 +2200,7 @@ impl SessionVars {
             &*CLUSTER,
             &CLUSTER_REPLICA,
             &*DATABASE,
+            &*SEARCH_PATH,
```

Comment:
Can you adjust the comment on line 2196 as well?

Reply:
Good catch. Adjusted :+1: